nautobot/extras/utils.py

```diff
@@ -446,6 +437,33 @@ def get_job_queue_worker_count(request=None, job_queue=None):
     return celery_queues.get(queue, 0)
 
 
+def get_job_queue(job_queue):
+    """
+    Search for a JobQueue instance based on the str job_queue.
+    If no existing Job Queue not found, return the default JobQueue with name CELERY_TASK_DEFAULT_QUEUE.
+    """
+    from nautobot.extras.models import JobQueue
+
+    queue = None
+    if is_uuid(job_queue):
+        try:
+            # check if the string passed in is a valid UUID
+            queue = JobQueue.objects.get(pk=job_queue)
+        except JobQueue.DoesNotExist:
+            queue, _ = JobQueue.objects.get_or_create(
+                name=settings.CELERY_TASK_DEFAULT_QUEUE, defaults={"queue_type": JobQueueTypeChoices.TYPE_CELERY}
+            )
```

Comment:
This addition feels weird to me - so if I call `get_job_queue("some_nonexistent_name")`, I get the default Celery queue as a response instead of `None`?

Reply:
Feels like it'd be better to just check for `queue is not None`?